client/src/components/profile/Comment.jsx

```diff
@@ -1,52 +1,52 @@
 import React, { useEffect, useState } from 'react';
-import axios from 'axios';
 import './Profile.css';
 import { Button } from 'react-bootstrap'
+import { useSelector } from "react-redux"
 
-const Comment = () => {
-    const [comment, setComment] = useState('');
+const Comment = ({ post, setCommentFlag }) => {
+  const [comment, setComment] = useState('');
 
-// handle change fn
-// handle submit fn
-const submitComment = () => {
-  // post on b/e routes
- axios.post('http://localhost:5000/comments', 
- {comment_content:comment //dbname: clname
-}).then(() => {
-  alert('successful insert');
-})
+  const userId = useSelector(state => state.user.user_id)
 
-};
+  const [comments, setComments] = useState([])
 
-const commentData = async () => {
-        const res = await axios.post("https://jsonplaceholder.typicode.com/posts", {comment})
-        .then( res => {
-          console.log(res.data);
-          console.log(res);
-        });
-};
+  const submitComment = () => {
+    // post on b/e routes
+
+    setCommentFlag(state => !state)
```

Comment:
=> for enter key
const submitComment = (event) => {
    // post on b/e routes
    event.preventDefault();

Reply:
:heavy_check_mark:  done